src/engine/ui/widgets/Combobox.hxx

```diff
@@ -31,35 +39,50 @@ public:
   * @return selected ID
   */
   int getActiveID() const { return m_activeID; };
+
   /** @brief Set ID of selected element
   * sets the ID of the selected element in the comboBox
   * @param ID the value to set the ID to
   */
   void setActiveID(int ID);
 
-  size_t count() const { return m_textField->count(); };
+  size_t count() const { return m_items.size(); };
+
+  /// the text displayed in the Combobox
   std::string activeText;
+  // this could probably be private tbh
+  bool centerText = true;
```

Comment:
Instead of using a bool to center the text, you should rather remove the variable and use your newly introduced enum TextFieldAlignment instead